fe1-web/components/BackButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Ionicons } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+import { Pressable, View } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+/**
+ * UI button to go back in navigation. You can specify the string corresponding to where it should
+ * navigate along with its size in its properties.
+ */
+const BackButton = (props: IPropTypes) => {
+  const { navigationTabName, size, testID } = props;
+  const navigation = useNavigation();
+
+  return (
+    // The view is there to set the button's clickable layout correctly.
+    <View style={{ width: size }}>
+      <Pressable onPress={() => navigation.navigate(navigationTabName as never)} testID={testID}>
```

Comment:
how come `as never` ? We do expect `navigationTabName` to occur

Reply:
Actually I don't really know why, but the IDE tells me that he has trouble converting the string to a `never`. So typing `as never` erases this red underline. There might be another way to deal with it though, I'll search for it on the web.